# [gg]
# Run st with another terminal such as $ st, then run nvim on that terminal and
# exit nvim. When the main terminal checked, there will be unknown errors, this
# patch removes reporting them. This is useful since "setsid st ${pwd}" and
# using both terminals become unusable due to error spam.

diff --git a/st.c b/st.c
index ebdf360..9606703 100644
--- a/st.c
+++ b/st.c
@@ -1604,12 +1602,10 @@ csihandle(void)
 {
 	char buf[40];
 	int len;

 	switch (csiescseq.mode[0]) {
 	default:
 	unknown:
-		fprintf(stderr, "erresc: unknown csi ");
-		csidump();
 		/* die(""); */
 		break;
 	case '@': /* ICH -- Insert <n> blank char */
@@ -1909,9 +1906,6 @@ strhandle(void)
 	case '^': /* PM -- Privacy Message */
 		return;
 	}
-
-	fprintf(stderr, "erresc: unknown str ");
-	strdump();
 }

 void
